playground/cdk/CdkSample.AppHost/appConfig.module.bicep

```diff
@@ -0,0 +1,34 @@
+targetScope = 'resourceGroup'
+
+@description('')
+param location string = resourceGroup().location
+
+@description('')
+param principalId string
+
+@description('')
+param principalType string
+
+
+resource appConfigurationStore_j2IqAZkBh 'Microsoft.AppConfiguration/configurationStores@2023-03-01' = {
+  name: toLower(take(concat('appConfig', uniqueString(resourceGroup().id)), 24))
+  location: location
+  sku: {
+    name: 'standard'
+  }
+  properties: {
+  }
+}
+
+resource roleAssignment_umUNaNdeG 'Microsoft.Authorization/roleAssignments@2022-04-01' = {
+  scope: appConfigurationStore_j2IqAZkBh
+  name: guid(appConfigurationStore_j2IqAZkBh.id, principalId, subscriptionResourceId('Microsoft.Authorization/roleDefinitions', '5ae67dd6-50cb-40e7-96ff-dc2bfa4b606b'))
+  properties: {
+    roleDefinitionId: subscriptionResourceId('Microsoft.Authorization/roleDefinitions', '5ae67dd6-50cb-40e7-96ff-dc2bfa4b606b')
+    principalId: principalId
+    principalType: principalType
+  }
+}
+
+output appConfigurationStore_j2IqAZkBh_endpoint string = appConfigurationStore_j2IqAZkBh.properties.endpoint
```

Comment:
@JoshLove-msft this needs a fix on the CDK side. We shouldn't emit this one by default.